CODEOWNERS

```
@@ -1 +1,3 @@
+@tchojnacki @jakubzehner @mlodybercik @piterek130 
```

```suggestion
* @tchojnacki @jakubzehner @mlodybercik @piterek130 
```

The current version sets the owners of a nonexistent `@tchojnacki` directory to `@jakubzehner @mlodybercik @piterek`.